complaints/templates/complaints/base.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+{% load static %}
+<html lang="en">
+   <head>
+      <title>CCIT BPHC</title>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+      <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css">
+      <script src="{% static 'main.js' %}"></script>
+      <link rel="stylesheet" href="{% static 'main.css' %}">
+   </head>
+   <body>
+      <style>.navbar {border-radius:0px;}</style>
+<nav class="navbar navbar-inverse">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>                        
+      </button>
+	  <img class="navbar-brand" src="/static/bitslogo.png"></img>
+      <a class="navbar-brand" href="#">CCIT BPHC</a>
+    </div>
+    <div class="collapse navbar-collapse" id="myNavbar">
+    <ul class="nav navbar-nav">
+	{% block content %}
+	{%endblock%}
```

Comment:
tags are not balanced. Where's </body>, </html> etc?

Reply:
testing testing

<div style="position: absolute; margin: 50px; padding: 50px; size: 20em"> <b> Testing whether github has a bug </b> </div>

Reply:
<form method="post"> 
    <input type="textbox" placeholder="bug">
<input type="submit" value="submit"> Submit</input>
</form>

Reply:
Can't balance all the tags in the base.html ,they are balanced in the respective pages.

Reply:
Why?

Reply:
because the code inside these tags varies in each page.

Reply:
And why does that mean that you cannot close an open element?